pyxem/utils/expt_utils.py

```diff
@@ -575,11 +586,17 @@ def find_beam_center_blur(z, sigma):
     Returns
     -------
     center : np.array
-        np.array [y, x] containing indices of estimated direct beam positon.
+        np.array [x, y] containing indices of estimated direct beam positon.
```

Comment:
is this a documentation change, or a behaviour change? Does it have to be like this?

Reply:
I didn't functionally change the code, only made it capable of handling cupy arrays. I think this was simply a mistake in the documentation -> the `[::-1]` reverses the indices, so the output will be `[x, y]` not `[y,x]`

Reply:
simple, will mark as resolved